### PR TITLE
Issue#16: Ability to add audience as array of strings

### PR DESCRIPTION
**Pull request by UmamaheshMaxwell**
Ability to add audience as array of strings

### CODE REVIEW

File: src/audience-handler.spec.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ describe('jwtAudienceHandler', () => {
   } as any;
 
   const options: IAuthAudienceOptions = {
-    audience: 'testAudience',
+    audience: ['testAudience1', 'testAudience2'],
```
Comment (candleshine) [inline src/audience-handler.spec.ts]: We need to handle both cases.  Strings and Arrays

File: src/audience-handler.spec.ts [hunk 2]
```diff
@@ -140,7 +140,7 @@ describe('jwtAudienceHandler', () => {
 
     const body = (result as any).body;
 
-    expect(body.jwt.aud).toBe(payload.aud);
+    expect(body.jwt.aud).toEqual(payload.aud);
```
Comment (candleshine) [inline src/audience-handler.spec.ts:151]: An array is going into the payload.  That should not be the case. there should be only one element of the array going into the payload .

**Review by candleshine — APPROVED** [review]
I will add to this. thanks for the test

**Review by etsuo — CHANGES_REQUESTED** [review]
needs to be properly rebased

**Pull request closed without merging**